pkg/okteto/k8s.go

```diff
@@ -19,6 +18,11 @@ import (
 var timeout time.Duration
 var tOnce sync.Once
 
+const (
+	// oktetoKubernetesTimeoutEnvVar defines the timeout for kubernetes operations
+	oktetoKubernetesTimeoutEnvVar = "OKTETO_KUBERNETES_TIMEOUT"
```

Comment:
would it make sense to move this into the k8s pkg, at a file called constants under this module? and export it? same for "job" and "deployment" ?

Reply:
I think that since the only place this envvar is being used is here, this package is responsible from handling it.

I think that if we force to group these under a package, we'll end up with the same problem of big hard to manage packages